AdminWebsite/AdminWebsite/Program.cs

```diff
@@ -18,12 +19,23 @@ private static IHostBuilder CreateWebHostBuilder(string[] args)
         {
             const string vhInfraCore = "/mnt/secrets/vh-infra-core";
```

Comment:
Do we need these two lines if we are looping through them all now?

Reply:
good point no.